src/inference/CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ file (GLOB LIBRARY_SRC
         ${CMAKE_CURRENT_SOURCE_DIR}/src/cpp_interfaces/interface/*.cpp
       )
 
-set(OV_STATIC_DEPENDENT_FILES ${CMAKE_CURRENT_SOURCE_DIR}/src/file_utils.cpp)
 list(REMOVE_ITEM LIBRARY_SRC ${OV_STATIC_DEPENDENT_FILES})
```

Comment:
Maybe we also need remove this line?

Reply:
not sure, but maybe we need to move this conditions to `openvino::util` itself.
let's see what tests result show us